key-manager/src/main/scala/org/bitcoins/keymanager/WalletStorage.scala

```diff
@@ -111,13 +114,22 @@ object WalletStorage {
 
     val readJsonTupleEither: CompatEither[
       ReadMnemonicError,
-      (String, String, String)] = jsonE.flatMap { json =>
+      (String, String, String, Long)] = jsonE.flatMap { json =>
       logger.trace(s"Read encrypted mnemonic JSON: $json")
+      val creationTimeNum = Try(json(CREATION_TIME).num.toLong) match {
+        case Success(value) =>
+          value
+        case Failure(err) if err.isInstanceOf[NoSuchElementException] =>
+          // If no CREATION_TIME is set, we set date to start of bitcoin-s wallet proj
+          // default is Block 555,990 block time 2018-12-28
+          1546042867L
```

Comment:
Can we put this into some util file with a nice name? I don't really like this magic number, and we may want to use it else where.